### PR TITLE
DDCE-487:updated BGV Color

### PR DESCRIPTION
**Pull request by gayathri-rajagopalan**
# DDCE-487

##updated Cosmetic Feature - Background Color

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

## Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

## Checklist PR Reviewer
 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/assets/stylesheets/lisa-app.scss [hunk 1]
```diff
@@ -52,4 +56,3 @@ p.indent {
 
 #whyGiveThisRating {
     min-height: 150px;
-}
```
Comment (DannyPennington97) [inline app/assets/stylesheets/lisa-app.scss:55]: Can you re-add this closing bracket please

**Comment by DannyPennington97** [comment]
Also I think you need to do a rebase as you are missing a commit causing the acceptance tests to fail on your branch currently

File: app/assets/stylesheets/lisa-app.scss (hunk 1 above)
Comment (gayathri-rajagopalan) [reply app/assets/stylesheets/lisa-app.scss:55]: Added the braces and squashed commit
Thanks

**Comment by Ufuoma-Daniel** [comment]
Friendly reminder, you don't need to mention you squashed your commit in the commit message. The commit message should always describe the changes you've made within that commit.

**Comment by gayathri-rajagopalan** [comment]
> Also I think you need to do a rebase as you are missing a commit causing the acceptance tests to fail on your branch currently

--Updated my branch with the latest master and correct commit message.

**Comment by DannyPennington97** [comment]
Nice one, everything now passing. Change looks good

**Pull request merged**